src/Illuminate/Mail/Mailer.php

```diff
@@ -383,7 +383,7 @@ protected function parseView($view)
      * Send a Swift Message instance.
      *
      * @param  \Swift_Message  $message
-     * @return void
+     * @return int|null
```

Comment:
Regardless what ever swift message return, `@return void` indicate that it should be ignored/not to depend on the return value (as mention by @GrahamCampbell)